runtime/doc/develop.txt

```diff
@@ -102,11 +102,11 @@ Examples:
 The provider framework invokes Vimscript from C.  It is composed of two
 functions in eval.c:
 
-- eval_call_provider(name, method, arguments, discard): calls
-  provider#{name}#Call with the method and arguments. If discard is true, any
+- eval_call_provider({name}, {method}, {arguments}, {discard}): Calls
+  provider {name} with {method} and {arguments}. If {discard} is true, any
```

Comment:
No, `provider#{name}#Call` is something that indeed exists and this was intentional. Like `provider#ruby#Call`, `provider#python3#Call`.

Reply:
ah whoops, I forget that vimscript syntax. Okay, I think it should just be wrapped in backticks then for clarity, and the arguments should be wrapped in brackets